lib/maths/CTimeSeriesDecompositionDetail.cc

```diff
@@ -270,6 +270,10 @@ const std::string VERSION_6_3_TAG("6.3");
 const std::string VERSION_6_4_TAG("6.4");
 
 // Periodicity Test Tags
+// Version 7.9
+const core::TPersistenceTag SHORT_WINDOW_7_9_TAG{"e", "short_window"};
+const core::TPersistenceTag LONG_WINDOW_7_9_TAG{"f", "long_window"};
```

Comment:
I think it's a bad idea that the long names here are identical to the ones used in `SHORT_WINDOW_6_3_TAG` and `LONG_WINDOW_6_3_TAG`.  It may not cause a problem immediately, but if the model visualizer is taken further it might in the future.

Reply:
See [93cbad3](https://github.com/elastic/ml-cpp/pull/1297/commits/93cbad37cbef56122803d5743d9ef6989eef8fd0).